src/main/java/org/opentripplanner/graph_builder/module/OsmBoardingLocationsModule.java

```diff
@@ -185,15 +186,15 @@ private boolean connectVertexToStop(TransitStopVertex ts, StreetIndex index) {
 
   private StreetEdge linkBoardingLocationToStreetNetwork(StreetVertex from, StreetVertex to) {
     var line = GeometryUtils.makeLineString(List.of(from.getCoordinate(), to.getCoordinate()));
-    return StreetEdge.createStreetEdge(
-      from,
-      to,
-      line,
-      new LocalizedString("name.platform"),
-      SphericalDistanceLibrary.length(line),
-      StreetTraversalPermission.PEDESTRIAN_AND_BICYCLE,
-      false
-    );
+    return new StreetEdgeBuilder<>()
+      .withFromVertex(from)
+      .withToVertex(to)
+      .withGeometry(line)
+      .withName(new LocalizedString("name.platform"))
```

Comment:
The `new LocalizedString("name.platform")` can be a constant.

Reply:
done